diff --git a/node_modules/@mui/x-data-grid/esm/hooks/features/pagination/gridPaginationUtils.js b/node_modules/@mui/x-data-grid/esm/hooks/features/pagination/gridPaginationUtils.js
index e115672..57f0f1f 100644
--- a/node_modules/@mui/x-data-grid/esm/hooks/features/pagination/gridPaginationUtils.js
+++ b/node_modules/@mui/x-data-grid/esm/hooks/features/pagination/gridPaginationUtils.js
@@ -1,5 +1,5 @@
 import { GridSignature } from "../../../constants/signature.js";
-const MAX_PAGE_SIZE = 100;
+const MAX_PAGE_SIZE = 100000;
 export const defaultPageSize = autoPageSize => autoPageSize ? 0 : 100;
 export const getPageCount = (rowCount, pageSize, page) => {
   if (pageSize > 0 && rowCount > 0) {
diff --git a/node_modules/@mui/x-data-grid/hooks/features/pagination/gridPaginationUtils.js b/node_modules/@mui/x-data-grid/hooks/features/pagination/gridPaginationUtils.js
index d64370d..a985afd 100644
--- a/node_modules/@mui/x-data-grid/hooks/features/pagination/gridPaginationUtils.js
+++ b/node_modules/@mui/x-data-grid/hooks/features/pagination/gridPaginationUtils.js
@@ -5,7 +5,7 @@ Object.defineProperty(exports, "__esModule", {
 });
 exports.throwIfPageSizeExceedsTheLimit = exports.getValidPage = exports.getPageCount = exports.getDefaultGridPaginationModel = exports.defaultPageSize = void 0;
 var _signature = require("../../../constants/signature");
-const MAX_PAGE_SIZE = 100;
+const MAX_PAGE_SIZE = 100000;
 const defaultPageSize = autoPageSize => autoPageSize ? 0 : 100;
 exports.defaultPageSize = defaultPageSize;
 const getPageCount = (rowCount, pageSize, page) => {
